doc/source/parameters/resolved_cfd-dem/resolved_cfd-dem.rst

```diff
@@ -159,11 +165,29 @@ The following parameter and subsection are all inside the subsection ``particle
 
 * The subsection ``omega`` defines the initial value of the particle rotational velocity if the parameter ``integrate motion=true``. Otherwise, it defines the particle's rotational velocity at all times. This rotational velocity is expressed as a function that can evolve in time. Each component of the ``Function expression`` corresponds to the value of its component in the X, Y, and Z direction. It's important to note that even the 2D solver uses the rotational velocity in 3D. In that case, it will only use the Z component of the rotational velocity.
 
+* The subsection ``orientation`` defines the initial value of the particle angular position around each of the axes X, then Y and then Z. This angular position is expressed as a function that can evolve in time. It's important to note that even the 2D solver uses the rotational velocity in 3D. In that case, it will only use the Z component of the rotational velocity, but all three should be defined.
+
 * The ``inertia`` parameter is used to define one of the diagonal elements of the rotational inertia matrix. Since we are defining spherical particles, we assume a uniform distribution of mass, and as such, all the diagonal elements of the rotational inertia matrix are the same.
 
 * The ``pressure x``, ``pressure y``, and ``pressure z`` parameters are used to define the X, Y, and Z coordinate offset of the pressure reference point relative to the center of the particle. These parameters are used when the ``assemble Navier-Stokes inside particles`` parameter is set to true to define the pressure reference point.
 
-* The ``radius`` parameter is used to define the radius of this particle.
+* The ``type`` parameter is used to define the geometry type of the particle. The alternatives in 2D are: sphere, ellipsoid, rectangle. Additional alternatives include: cone, death star, cut hollow sphere, torus.
+
+* The subsection ``solid arguments`` is used to define the solid parameters in the form of a tri-components function. The arguments, which must be written in triplets, are:
+
+    * Sphere: radius, [none], [none];
+
+    * Rectangle: x half length, y half length, z half length;
+
+    * Ellipsoid: x radius, y radius, z radius;
+
+    * Torus: torus radius, torus thickness radius, [none];
+
+    * Cone: tan(base angle), height, [none];
+
+    * Cut Hollow Sphere: radius, cut height, wall thickness;
+
+    * Death Star: sphere radius, hole radius, distance between centers.
 
```

Comment:
maybe there should be a note saying : since it needs to be a triplet, you may add any integer value in the [none] places but they won't be used.

Reply:
I know this implies a bit of work, but maybe it would be clearer to use variable names and figures with the shapes to show the parameters?

Reply:
@CaroleAnneDaunais That's going to get complicated real quick. I think it's best to resort to something like the dealii grid generator where a string list is parsed and we follow the documentation to know the arguments.
We can keep examples for multiples shapes to illustrate the principle.